packages/dumb/src/settings/diabeloop.css

```diff
@@ -0,0 +1,102 @@
+.copyButton {
```

Comment:
Some classes are not used because the components using them are yet to be migrated. If I remove them, they will have to be added once we migrate the other components. Do you still prefer to remove them?

If I rename the file, we are at risk of duplicating it when migrating other component using this file (if the migration is done by someone not know that renamedFile.css is in fact diabeloop.css, or if the information is forgotten). Do you still prefer to rename the file?

Reply:
ok I didn't see that older components still uses theses classes. So no problem to keep them.
I understand the migration strategy, keep this like that ;)